Use a separate name for encoder layer fastpath

mikekgfb opened the pull request:
Summary: Use a separate name for encoder layer fastpath

Test Plan: sandcastle

Differential Revision: D38318075

jbschlosser reviewed (approved):
LGTM! (one lint issue)

Commit:
Use a separate name for encoder layer fastpath (#82579)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/82579

Use a separate name for encoder layer fastpath

Test Plan: sandcastle

Reviewed By: jbschlosser

Differential Revision: D38318075

fbshipit-source-id: 81ec9714c01c39f4d5aadbdc725b64f236fd5894
